gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/mapper/ConfigurationSerializationMapper.java

```diff
@@ -52,6 +53,23 @@ default String serializeConfiguration(Object configuration) {
         }
     }
 
+    @Named("serializeConfigMap")
+    default Map<String, Object> serializeConfigMap(Object configuration) {
```

Comment:
maybe convert would be a better name as we don't actually 'serialise' here WDYT ? this way we could name it `convertConfig` and avoid the confusion with k8s `ConfigMap`.

also, just curious, why aren't we just performing a cast here ?

as a side note, also wondering why the Object mapper is not define as a static field.